_posts/en/newsletters/2019-10-23-newsletter.md

```diff
@@ -0,0 +1,127 @@
+---
+title: 'Bitcoin Optech Newsletter #69'
+permalink: /en/newsletters/2019/10/23/
+name: 2019-10-23-newsletter
+slug: 2019-10-23-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter requests testing of the C-Lightning
+and Bitcoin Core release candidates, invites
+participation in a taproot review club, highlights updates to two
+Bitcoin wallets, and describes a few notable changes to popular
+Bitcoin infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Help test release candidates:** experienced users are encouraged to
+  help test the latest release candidates for the upcoming versions of
+  [Bitcoin Core][Bitcoin Core 0.19.0] and [C-Lightning][c-lightning
+  0.7.3].
+
+## News
+
+- **Taproot review club:** starting the first week of November, several Bitcoin
```

Comment:
Semantic nit: in the [Square Crypto proposal](https://docs.google.com/document/d/1G48-yhZMLLMZW68Bq59h_FBi__pwFKoWKF1D385I38Y/edit#heading=h.csveiwpa8v3c), no mention is made of a club and there could be confusion with the PR review club like [here](http://www.erisian.com.au/meetbot/bitcoin-core-dev/2019/bitcoin-core-dev.2019-10-17-19.00.log.html#l-160). Suggest: s/Taproot review club/Taproot BIP review/

Reply:
The [overview](https://github.com/ajtowns/taproot-review) has a section entitled "What happens this review club is over".  Also, the idea was originally pitched as "BIP-proposal-review-club".

Reply:
Do you think there's harm in calling it a review club?  I didn't think so myself, as any confusion seems likely to me to be a chance to promote both organizations.

Reply:
Oh ok, we're not looking at the same doc. Yes, I would avoid confusion between the two unless it's expressly desired to extend the concept, but it's not my call.

Reply:
TIL about aj's doc. Interestingly it does not have any dates for the sessions, apart from the Oct. 30 signup, whereas moneyball's doc launches the sessions on November 5th.

Reply:
I pinged the people involved and will follow up on this.  Thanks for raising this point!

Reply:
I also created a PR to add the starting date https://github.com/ajtowns/taproot-review/pull/2

Reply:
and finally, a suggested renaming back to the original name https://github.com/ajtowns/taproot-review/pull/3